content/00.build/30.zksync-sso/10.architecture.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Architecture
+description: Learn about architectural components of ZKsync SSO and how they work together.
+---
+
+The ZKsync SSO consists of three components that work together to offer a seamless experience for both developers and users.
+
+## ZKsync SSO Developer SDK
+
+The **Developer SDK** is a client-side library that developers can integrate into their applications with minimal effort.
+It handles authentication, session management, and spend limit controls,
```

Comment:
throughout the doc, sessions and spend limits are presented as two separate things, whereas spend limits are just one use case of sessions.

Reply:
I updated this so it's more clear that you can have sessions and spend limits. While currently we require sessions to have spend limits, you aren't really required to have spend limits on your sessions!